exercises/practice/leap/.approaches/date-addition-chrono/content.md

```diff
@@ -0,0 +1,29 @@
+# `chrono::Date` addition
+
+```rust
+use chrono::{prelude::*, Duration};
+
+pub fn is_leap_year(year: u64) -> bool {
+    (Utc.ymd(year as i32, 2, 28) + Duration::days(1)).day() == 29
+}
+```
+
+This approach may be considered a "cheat" for this exercise.
```

Comment:
This might be good content for a caution block:

````suggestion
```exercism/caution
This approach may be considered a "cheat" for this exercise.
```
````

Reply:
I don't think we did that elsewhere for a "cheat", but I think it's okay to make it a caution.